code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -666,30 +686,28 @@ nobiliumsuppression = INFINITY
 /datum/gas_reaction/stimformation/init_reqs()
 	min_requirements = list(
 		/datum/gas/tritium = 30,
-		/datum/gas/plasma = 10,
 		/datum/gas/bz = 20,
 		/datum/gas/nitryl = 30,
-		"TEMP" = STIMULUM_HEAT_SCALE/2)
+		"TEMP" = 1500)
 
 /datum/gas_reaction/stimformation/react(datum/gas_mixture/air)
 	var/list/cached_gases = air.gases
 
 	var/old_heat_capacity = air.heat_capacity()
-	var/heat_scale = min(air.temperature/STIMULUM_HEAT_SCALE,cached_gases[/datum/gas/tritium][MOLES],cached_gases[/datum/gas/plasma][MOLES],cached_gases[/datum/gas/nitryl][MOLES])
-	var/stim_energy_change = heat_scale + STIMULUM_FIRST_RISE*(heat_scale**2) - STIMULUM_FIRST_DROP*(heat_scale**3) + STIMULUM_SECOND_RISE*(heat_scale**4) - STIMULUM_ABSOLUTE_DROP*(heat_scale**5)
+	var/heat_scale = min(air.temperature / 2500, cached_gases[/datum/gas/tritium][MOLES], cached_gases[/datum/gas/nitryl][MOLES])
```

Comment:
why remove stim_heat_scale?

Reply:
mm i was tryong to do some testing, it seems i missed to restore this one. thanks for noticing